### DISCUSSION TITLE
Jetbrains IDE integration

### DISCUSSION THREAD

**Discussion by bric3** [excalidraw/excalidraw#3766] [General]
Hi there,

I worked on an integration of excalidraw within Jetbrains IDEs (Golang, CLion, IntelliJ, WebStorm, etc.)
I just published the first EAP, that you need need to install from disk while the plug-in approval is pending.

Many thanks to @ad1992 who illuminated me on my JS/React questions, I’m a pure novice on the topic.

Release: https://github.com/bric3/excalidraw-jetbrains-plugin/releases/tag/v0.1.0-eap

**Comment by ad1992** [comment]
That's awesome @bric3, congratulations 🎉 , glad I could help :).

**Reply by bric3** [reply]
You happened to be a key factor in the success of this integration. Thank you!

**Comment by faisal6621** [comment]
Good work @bric3. I hope that plugin could be treated as official one.
